server/parsec/components/account.py

```diff
@@ -96,6 +96,23 @@ class BaseAccountComponent:
     def __init__(self, config: BackendConfig):
         self._config = config
 
+    async def get_password_algorithm_or_fake_it(self, email: EmailAddress) -> PasswordAlgorithm:
```

Comment:
I would remove the `or_fake_it` from the name and add more detail in docstring (see below)
```suggestion
    async def get_password_algorithm(self, email: EmailAddress) -> PasswordAlgorithm:
```

Reply:
I'd rather keep the naming as-is since it is very exotic to return dummy data instead of returrning `None` or raising an exception